src/cuda/mod.rs

```diff
@@ -45,7 +45,8 @@ pub struct Device {
     compute_capability: (u32, u32),
     pci_id: PciId,
     uuid: Option<DeviceUuid>,
-    device: rustacuda::device::Device,
+    /// Handle to Cuda device
+    pub device: rustacuda::device::Device,
```

Comment:
Why does it need to be public now?

Reply:
Clippy was throwing an error on this one, reporting "device" was never read.  I'll see if this still occurs with earlier Rust compiler.